### DISCUSSION TITLE
Generating empty values in Yaml

### DISCUSSION THREAD

**Discussion by tetsuya28** [cue-lang/cue#3383] [Q&A]
## 
I want to generate the following yaml for OpenTelemetry Collector.
`exporters.googlemanagedprometheus` has empty value, not `null` nor `{}`.
How to write a cue for this situation.

```yaml
exporters:
  googlemanagedprometheus:
```

## Cue version
```
cue version v0.6.0

go version go1.20.6
      -buildmode exe
       -compiler gc
       -trimpath true
     CGO_ENABLED 0
          GOARCH arm64
            GOOS darwin
```

**Comment by mvdan** [comment]
Why exactly do you need to encode YAML like that? Such empty or missing values in YAML are equivalent to `null`, so you can encode the following and it's exactly the same value in YAML:

```
exporters:
  googlemanagedprometheus: null
```

CUE has `null` as well, which you can use for this purpose.

I'm not opposed to supporting this in some way per se, but I also would like to understand why it matters.
